src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -544,6 +544,14 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     void TermControl::_MouseClickHandler(Windows::Foundation::IInspectable const& /*sender*/,
                                          Input::PointerRoutedEventArgs const& args)
     {
+        // Ignore mouse events while the terminal does not have focus. 
+        // This prevents the user from selecting and copying text if they 
+        // click inside the current tab to refocus the terminal window. 
+        if (!_focused) {

```

Comment:
nit: our code style has braces on the next line

Reply:
Fixed in commit mblowey/terminal@f386d058d81ab10e3a35a5a2bc0da5ae227d78fb.